kafka/monitoring-kafka-performance-metrics.md

```diff
@@ -327,14 +328,14 @@ Thanks to [Gwen Shapira][gwenshap], System Architect at Confluent for generously
 [choose-topics]: http://www.confluent.io/blog/how-to-choose-the-number-of-topicspartitions-in-a-kafka-cluster/
 [consumer-group-example]: https://cwiki.apache.org/confluence/display/KAFKA/Consumer+Group+Example
 [cpu-bottleneck]: https://cwiki.apache.org/confluence/display/KAFKA/Operations#Operations-Hardware
-[design-motivation]: https://kafka.apache.org/08/design.html
+[design-motivation]: https://kafka.apache.org/20/documentation/#design
```

Comment:
any reason we need to link to the versioned docs? Seems like the links would be more durable like so:

https://kafka.apache.org/documentation/#design

Reply:
Ack